AIPscan/Aggregator/mets_parse_helpers.py

```diff
@@ -49,31 +49,32 @@ def get_aip_original_name(mets):
     # Negated as we're going to want to remove this length of values.
     NAMESUFFIX = -len("-00000000-0000-0000-0000-000000000000")
 
+    # Other intellectual entities exist in the METS, i.e. for empty
+    # directories. We can identify those with an invalid name prefix.
```

Comment:
It might be worth adding a comment that this name prefix is an indicator of an invalid `<premis:originalName>` value specifically within the context of a dmdSec. I was confused for a minute because the `<premis:originalName>` value in the `amdSec` of an original file also begins with "%transferDirectory%".